### PR TITLE
solve issue #78

### PR DESCRIPTION
**Pull request by amitYadavDev**
Screenshot_20200112-194413
**Description (required)**

Fixes #{GitHub issue number} {Github issue title}

What changes did you make and why?

**Tests performed (required)**

Tested {build variant, e.g. ProdDebug} on {name of device or emulator} with API level {API level}.

**Screenshots showing what changed (optional - for UI changes)**

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

**Comment by amitYadavDev** [comment]
@jrandallclark  can i make a slack channel for this community and share link with you?

File: app/src/main/java/memphis/myapplication/UI/SettingsFragment.java [hunk 1]
```diff
@@ -58,8 +76,24 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
...
             @Override
             public void onChanged(Uri uri) {
-                Picasso.get().load(uri).placeholder(R.drawable.avatar).memoryPolicy(MemoryPolicy.NO_CACHE).fit().centerCrop().into(m_imageView);
+
+
+               Picasso.get().load(uri).placeholder(R.drawable.avatar).memoryPolicy(MemoryPolicy.NO_CACHE).fit().centerCrop().into(m_imageView);
                 setupToolbar(uri);
+
+                 //       Picasso.get().load(uri).placeholder(R.drawable.avatar).resize(120, 120).centerCrop()
+               //             .into(m_imageView);
```
Comment (jrandallclark) [inline app/src/main/java/memphis/myapplication/UI/SettingsFragment.java]: Remove any commented out code.

File: app/src/main/java/memphis/myapplication/UI/SettingsFragment.java [hunk 2]
```diff
@@ -117,4 +151,50 @@ private void setupToolbar(Uri uri) {
         Toolbar toolbar = toolbarHelper.setupToolbar(uri);
         ((AppCompatActivity)getActivity()).setSupportActionBar(toolbar);
     }
+
+    private  Uri rotateImageIfRequired(Context context, Bitmap img, Uri selectedImage) {
+
+        // Detect rotation
+         int rotation = getRotation(context, selectedImage);
```
Comment (jrandallclark) [inline app/src/main/java/memphis/myapplication/UI/SettingsFragment.java:151]: Fix indentation.

File: app/src/main/java/memphis/myapplication/UI/SettingsFragment.java [hunk 3]
```diff
@@ -117,4 +151,50 @@ private void setupToolbar(Uri uri) {
...
+
+        int rotation = 0;
+        ContentResolver content = context.getContentResolver();
+
+        Cursor mediaCursor = content.query(MediaStore.Images.Media.EXTERNAL_CONTENT_URI,
+                new String[] { "orientation", "date_added" },
+                null, null, "date_added desc");
+
+        if (mediaCursor != null && mediaCursor.getCount() != 0) {
+            while(mediaCursor.moveToNext()){
```
Comment (jrandallclark) [inline app/src/main/java/memphis/myapplication/UI/SettingsFragment.java]: Space after "while".

File: app/build.gradle [hunk 4]
```diff
@@ -74,4 +74,5 @@ dependencies {
     // ViewModel and LiveData
     def lifecycle_version = "2.0.0"
     implementation "androidx.lifecycle:lifecycle-extensions:$lifecycle_version"
+
```
Comment (jrandallclark) [inline app/build.gradle]: Nothing was added here. Remove build.gradle from your commit.

**Review by jrandallclark — CHANGES_REQUESTED** [review]
Does this button rotate the photo 90 degrees each time you press it?

> @jrandallclark can i make a slack channel for this community and share link with you?

Our organizational Slack channel will open up soon.

**Comment by amitYadavDev** [comment]
@jrandallclark yeah, i fixed

**Comment by jrandallclark** [comment]
I don't think we want users to rotate their profile pictures within the app. The bug here is that the profile photo is sometimes displayed incorrectly regardless of the correct orientation of the photo. If a user wants to change the orientation, they should rotate it in an image editor and re-upload it to the app. The app should then display the photo correctly.

**Pull request closed without merging**